src/hooks/useURLParams.ts

```diff
@@ -0,0 +1,26 @@
+import { useState, useMemo } from 'react'
+
+function setURLFilterParams(filterString: string, moveNow?: boolean): string | void | boolean {
+  if (!window || !window.history) return false
+  const defaultLocation = window.location.href.split('?')[0]
```

Comment:
This will sometimes get hash, and other times it won't `...?search#hash` vs `...#hash?search`